slides/fear-of-wrong.md

```diff
@@ -1,3 +1,14 @@
 ##  Fear Of Wrong
 
 Don't fear.
+
+Note:
+Henning
+
+We always hear this phrase "use the right tool for the job". But how do I know which
+one is the right tool?
+There are certainly cases where this is true. You maybe don't want to build a simple
+landing page with a JavaScript front end framework, but for most cases the answer
+to this question is not necessarily a technical one.
+
```

Comment:
Also: all the information you need is out there and easily accessible, also there is never a 100% guarantee that the framework will be around for ever or will be the perfect choice for you so just loose the fear. It is a waste of time.